airflow/docker.go

```diff
@@ -54,6 +58,32 @@ type ComposeConfig struct {
 	AirflowWebserverPort string
 }
 
+// SettingsConfig is input data to generate connections, pools, and variables
```

Comment:
@benjamingregory could maybe break out any nested types into top-level code and just refer to those types as you create new types, like:

```
type SettingsConfig struct {
   Airflow: AirflowConfig
}
```
where the `AirflowConfig` is defined as a top-level type.

Could help avoid some pyramid of doom indentation.